src/views/SessionView/Whiteboard.vue

```diff
@@ -279,6 +280,7 @@ export default {
       autoPickTool: false,
       autoPickToolText: false,
       defaultBrushWidth: 5,
+      allowZoom: false,
```

Comment:
disable the pan tool's ability to zoom; all zooming is done ourselves via zwibbler api